lib/rubocop/cop/rspec/no_expectation_example.rb

```diff
@@ -61,11 +89,20 @@ def on_block(node)
           return unless regular_or_focused_example?(node)
           return if including_any_expectation?(node)
           return if including_any_skip_example?(node)
+          return if including_any_allowed_pattern?(node)
```

Comment:
I'd rather modify the including_any_expectation? method to account for the allowed patterns

Reply:
@Darhazer @pirj 
Fixed to extend `including_any_expectation?` How do you like it? Could you review it again?